packages/atomic/src/components/atomic-breadcrumb-manager/atomic-breadcrumb-manager.tsx

```diff
@@ -41,153 +48,160 @@ export class AtomicBreadcrumbManager implements InitializableComponent {
   @State() private collapsedBreadcrumbsState: string[] = [];
   @State() public error!: Error;
 
+  /**
+   * Number of breadcrumbs to be shown before collapsing.
+   */
   @Prop() public collapseThreshold = 5;
+  /**
+   * Character that divides each path segment in a category facet breadcrumb
+   */
   @Prop() public categoryDivider = '/';
 
+  @BindStateToI18n()
+  @State()
+  public strings: I18nState = {
+    breadcrumb: (variables) =>
+      this.bindings.i18n.t('removeFilterOn', variables),
+  };
+
   public initialize() {
     this.breadcrumbManager = buildBreadcrumbManager(this.bindings.engine);
   }
 
-  private get facetBreadcrumbs() {
-    return this.breadcrumbManagerState.facetBreadcrumbs.map((breadcrumb) => {
-      const breadcrumbsValues = this.getBreadrumbValues(breadcrumb);
-      return (
-        <ul part="breadcrumbs">
-          <li part="breadcrumb-label">{breadcrumb.field}:&nbsp;</li>
-          {breadcrumbsValues}
-        </ul>
-      );
-    });
+  private getBreadcrumbValue(
+    value: string,
+    breadcrumbValue: BreadcrumbValue<BaseFacetValue> | CategoryFacetBreadcrumb
+  ) {
+    return (
+      <button
+        part="breadcrumb-button"
+        class="text-on-background-variant breadcrumb-button flex items-center"
+        aria-label={this.strings.breadcrumb({value})}
+        onClick={() =>
+          this.breadcrumbManager.deselectBreadcrumb(breadcrumbValue)
+        }
+      >
+        <span
+          part="breadcrumb-value-label"
+          class="whitespace-nowrap overflow-ellipsis overflow-hidden"
+        >
+          {value}
+        </span>
+        <div
+          part="breadcrumb-clear"
+          role="button"
+          class="breadcrumb-clear ml-1.5"
+          innerHTML={mainclear}
+        />
+      </button>
+    );
+  }
+
+  private getBreadcrumbWrapper(field: string, children: any) {
```

Comment:
Can we use a relevant type instead of any, is there a JSX related type?

Reply:
By grepping a bit around stencil documentation, I think this could be typed `VNode[]`